stories/DateRangePicker.js

```diff
@@ -83,7 +83,6 @@ storiesOf('DateRangePicker (DRP)', module)
     moment.locale('fa');
     return (
       <DateRangePickerWrapper
-        placeholder="تقویم فارسی"
```

Comment:
isn't the goal here to have farsi/arabic in the example? why remove the placeholder?

Reply:
This didn't actually do anything. `placeholder` is not a valid prop on the DRP and the text itself translates to Farsi Calendar which is not a meaningful value for the `startDatePlaceholderText` or the `endDatePlaceholderText` props.

Reply:
👍